### ISSUE TITLE
Spread feature to complement rest argument feature

### ISSUE THREAD

**Issue by StreetStrider** [less/less.js#2614]
Hello. I've found `@rest...` feature which allows to capture variadic arguments in a list. So I've tried to use it, but faced issue when I need to pass my variadic arguments to another mixin. Here's synthetic example:
```less
.x(@a; @b)
{
  a: @a;
  b: @b;
}
.x(wow; @rest...) // special form of `.x`
{
  c: something-special;
  .x(...@rest); // <- spread required here
}
a
{
  .x(wow; 1; 2);
}
```
So, in special form I capture all arguments and pass it to common form.

The only workaround for now is to pass all arguments manually. But this case becomes ugly when number of arguments grows and/or they obtain default values. So I need to take into account all defaults: extract them in variables and set they as defaults in both forms of mixin. `rest` feature is good, but it isn't complete without ability to spread captured list to another mixin. Thanks.

**Comment by seven-phases-max** [comment]
See #1857 (don't forget to put another +1 there).

**Comment by seven-phases-max** [comment]
Closing as duplicate of already implemented (but not merged) #1857.

**Comment by StreetStrider** [comment]
@seven-phases-max, thanks for heads up. And please sorry for dup, I've looked through issues by «spread» and «rest» but found nothing.
